@here/harp-mapview/lib/Tile.ts

```diff
@@ -759,12 +759,12 @@ export class Tile implements CachedResource {
     }
 
     /**
-     * Overrides the default value for [[hasGeometry]].
+     * Overrides the default value for [[hasGeometry]] if value is not `undefined`.
      *
      * @param value A new value for the [[hasGeometry]] flag.
      */
-    forceHasGeometry(value: boolean) {
-        if (value) {
+    forceHasGeometry(value: boolean | undefined) {
```

Comment:
a call to `Tile.forceHasGeometry()` will not force the tile to have geometry. This is a bit weird. Shouldn't undefined be handled like true?

Reply:
Somebody made forceHasGeometry() pretty complex, such that it uses some empty DecodedTile, please go look and see Mr. Architect ;)
I added undefined to make it possible to properly reset the value to "look into the actual geometry (tile.objects[]) and see if there is something there", this is new and does no harm.